### PR TITLE
Measure duration of docker commands called from orchestrator

### PR DESCRIPTION
**Pull request by petertrr**
* Measure time of `docker build` command. Will be useful when we need to tune performance in the future.

Related to #344

```
# HELP save_orchestrator_docker_build_seconds_max  
# TYPE save_orchestrator_docker_build_seconds_max gauge
save_orchestrator_docker_build_seconds_max{baseImage="openjdk:11",cmd="BuildImageCmd",} 13.381837036
# HELP save_orchestrator_docker_build_seconds  
# TYPE save_orchestrator_docker_build_seconds summary
save_orchestrator_docker_build_seconds_count{baseImage="openjdk:11",cmd="BuildImageCmd",} 1.0
save_orchestrator_docker_build_seconds_sum{baseImage="openjdk:11",cmd="BuildImageCmd",} 13.381837036
```

### CODE REVIEW

File: save-orchestrator/build.gradle.kts [hunk 1]
```diff
@@ -92,7 +92,3 @@ kotlin.sourceSets.getByName("main") {
 tasks.withType<KotlinCompile>().forEach {
     it.dependsOn(generateVersionFileTaskProvider)
 }
-
-tasks.withType<Test> {
```
Comment (kgevorkyan) [inline save-orchestrator/build.gradle.kts:96]: Why?

File: save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/docker/ContainerManager.kt [hunk 2]
```diff
@@ -148,13 +156,20 @@ class ContainerManager(private val settings: DockerSettings) {
...
         val buildImageResultCallback: BuildImageResultCallback = try {
-            dockerClient.buildImageCmd(dockerFile)
+            val buildCmd = dockerClient.buildImageCmd(dockerFile)
                 .withBaseDirectory(tmpDir)
                 .withTags(setOf(imageName))
                 .withExtraHosts(hostIp?.let {
                     setOf("host.docker.internal:$hostIp")
                 } ?: emptySet())
-                .start()
+            buildCmd.execTimed(meterRegistry, "save.orchestrator.docker.build", "baseImage", baseImage) { record ->
```
Comment (kgevorkyan) [inline save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/docker/ContainerManager.kt:165]: What if `buildCmd` will be an `emptySet`, which behavior we will see? I don't see any `isEmpty` like checks

File: save-orchestrator/build.gradle.kts (hunk 1 above)
Comment (petertrr) [reply save-orchestrator/build.gradle.kts:96]: I thought that it pollutes logs too much, on CI it's hard to find which test has failed among all those log lines. Do you find it useful? I usually navigate to HTML reports anyway, but that may be just me.

File: save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/docker/ContainerManager.kt (hunk 2 above)
Comment (petertrr) [reply save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/docker/ContainerManager.kt:165]: No, `emptySet` is a possible value for `extraHosts` parameter. `buildCmd` is always a docker command instance

Comment (kgevorkyan) [reply save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/docker/ContainerManager.kt:165]: Ok, didn't notice

File: save-orchestrator/build.gradle.kts (hunk 1 above)
Comment (kgevorkyan) [reply save-orchestrator/build.gradle.kts:96]: Usually I used it in course of development, however, yes, we've a got a lot of extra logs, so I can enable it manually if I need

**Review by kgevorkyan — APPROVED** [review]

**Pull request merged**